### ISSUE TITLE
Verbesserungsvorschlag/Wunsch: Abfahrer manuell stoppen

### ISSUE THREAD

**Issue by Experience95** [Stephan-S/FS22_AutoDrive#1019]
Wenn CP oder ein anderer Helfer einen Häcksler fährt, scheint AD als Abfahrer am Vorgewende einen Befehl zu bekommen um anzuhalten, damit der Häcksler wenden kann. Nach der Wende wird der Abfahrer dann wieder gerufen.
Wenn ich nun selbst den Häcksler steuere versucht AD beim Wenden weiterhin nebenher zu fahren, schafft es aber logischerweise nicht und es gibt ein wildes gekurve vom Abfahrer am Vorgewende und es dauert ewig bis der Abfahrer wieder neben dem Häcksler ist.
Ich würde mir also eine Taste wünschen, um den Abfahrer zu pausieren wenn ich am Vorgewende ankomme und eine Taste, mit dem ich den Abfahrer nach dem umdrehen wieder zu mir rufen kann. Letzteres ist mit "Abfahrer rufen" ja quasi schon vorhanden.

LG
Robin

**Comment by Iwan1803** [comment]
Rohr ein-/ausfahren müsste genau dieses Verhalten bewirken - zumindest bei Dreschern klappt das auf jeden Fall.

**Comment by Experience95** [comment]
@Iwan1803 Klar, das funktioniert, bzw ist auch der Workaround den ich verwendet habe, aber

**Comment by Experience95** [comment]
Ja schon klar, das ist auch der Workaround den ich benutzt habe, aber das ist beim Häcksler recht nervig, da dann auch der Häcksler und das Schneidwerk abgeschaltet wird etc. Die Funktion des "pausierens" auf ner Taste wäre halt einfacher.

**Comment by regs1980** [comment]
> Die Funktion des "pausierens" auf ner Taste wäre halt einfacher.

Was wäre da einfacher?

Du springst aus dem Häcksler in den AD Abfahrer, um ihn zu pausieren, dann wieder zurück in den Häcksler, um die Wende zu machen, danach wieder in den AD Abfahrer, um ihn aus dem Pause-Modus herauszuholen und dann wieder in den Häcksler.
 
Das verbraucht mehr Zeit, als das Rohr einzufahren und wieder ausfahren. 

Für mich ergibt eine pausen Einstellung keinen Sinn.
Gerade, weil der AD Abfahrer auch noch die Zeit braucht, zum Häcksler zu fahren und sich zu positionieren.

Schneller geht es, Rohr einfahren, Wende machen, sich richtig positionieren, Rohr ausfahren, warten auf den AD Abfahrer und dann Schneidwerk starten und los häckseln.
